test/words.test.js

```diff
@@ -0,0 +1,13 @@
+import words from "../src/words.js";
+
+test("words without regex", () => {
+    expect(words('fred, barney, & pebbles')).toBe(
+        ['fred', 'barney', 'pebbles']
+    );
+});
+
+test("words with regex", () => {
+    expect(words('fred, barney, & pebbles', /[^, ]+/g)).toBe(
+        ['fred', 'barney', '&', 'pebbles']
+    );
+});
```

Comment:
Using toIncludeAllMembers will make the tests pass

```suggestion
test("words without regex", () => {
    expect(words('fred, barney, & pebbles')).toIncludeAllMembers(
        ['fred', 'barney', 'pebbles']
    );
});

test("words with regex", () => {
    expect(words('fred, barney, & pebbles', /[^, ]+/g)).toIncludeAllMembers(
        ['fred', 'barney', '&', 'pebbles']
    );
});
```